lib/parseArguments.js

```diff
@@ -136,10 +136,48 @@ function parseArguments(args) {
                 group: 'Options: Ambient Occlusion',
                 nargs: 1,
                 type: 'string'
+            },
+            'texcomp.enable': {
+                default : false,
+                describe: 'Compress textures.',
+                group: 'Options: Texture Compression',
+                type: 'boolean'
+            },
+            'texcomp.format': {
```

Comment:
For the `texcomp.*` options, are there any links we can point folks to, e.g., the doc from the vendor who wrote the tool we are forwarding the parameters to.

These links could be in the README.md.

We'll also link to @bagnell's upcoming blog post.